src/Sylius/Bundle/AdminBundle/Resources/config/services/listener.xml

```diff
@@ -16,7 +16,7 @@
         <defaults public="true" />
 
         <service id="sylius.listener.shipment_ship" class="Sylius\Bundle\AdminBundle\EventListener\ShipmentShipListener">
-            <argument type="service" id="sylius.email_manager.shipment" />
+            <argument type="service" id="Sylius\Bundle\CoreBundle\Mailer\ShipmentEmailManagerInterface" />
```

Comment:
Isn't this a BC break?
If I've overridden the `sylius.email_manager.shipment` service, this will ignore it and always use the one from core. I know it's a listener but still, might be annoying.

As a side note, since all the managers are in core, shouldn't we add different services for both admin and shop, so they can be decorated/substituted independently?